2025-05-18 01:54:02,704 - DeeperHubClient - INFO - Tentando conectar a ws://localhost:4000/ws
2025-05-18 01:54:04,768 - DeeperHubClient - INFO - Conexão estabelecida com ws://localhost:4000/ws
2025-05-18 01:54:04,857 - DeeperHubClient - INFO - Resposta de sucesso: user.create.success [payload={'id': '7def24c9-f718-4bbd-a5bf-b8d1f5b19ac5', 'username': 'test_user_2524c2ab', 'email': 'test_2524c2ab@example.com', 'is_active': True, 'inserted_at': '2025-05-18T04:54:04.809000Z', 'updated_at': '2025-05-18T04:54:04.809000Z'}]
2025-05-18 01:54:04,957 - DeeperHubClient - INFO - Resposta de sucesso: user.list.success [payload={'count': 2, 'users': [{'id': '0835c858-2e05-4076-af8d-23a215dab4b2', 'username': 'romeu', 'email': 'as@asas.as', 'is_active': True, 'inserted_at': '2025-05-18T03:18:54.004000Z', 'updated_at': '2025-05-18T03:18:54.004000Z'}, {'id': '7def24c9-f718-4bbd-a5bf-b8d1f5b19ac5', 'username': 'test_user_2524c2ab', 'email': 'test_2524c2ab@example.com', 'is_active': True, 'inserted_at': '2025-05-18T04:54:04.809000Z', 'updated_at': '2025-05-18T04:54:04.809000Z'}]}]
2025-05-18 01:54:05,328 - DeeperHubClient - INFO - Resposta de sucesso: auth.login.success [payload={'session_id': 'ceed7676-6577-499e-b680-cfb272f3b2db', 'username': 'test_user_2524c2ab', 'remember_me': False, 'user_id': '7def24c9-f718-4bbd-a5bf-b8d1f5b19ac5', 'access_token': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJkZWVwZXJfaHViX3dlYnNvY2tldCIsImV4cCI6MTc0NzU0NzY0NSwiaWF0IjoxNzQ3NTQ0MDQ1LCJpc3MiOiJkZWVwZXJfaHViIiwianRpIjoiMzEwOGMzNnA5dWJwMWRhNDNjMDAwMDgzIiwibmJmIjoxNzQ3NTQ0MDQ1LCJ0eXAiOiJhY2Nlc3MiLCJ1c2VyX2lkIjpudWxsfQ.TYY5_5Huwnp4qS28Lk4_3OL2s6gDVdJZvelMVcvNNSg', 'refresh_token': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJkZWVwZXJfaHViX3dlYnNvY2tldCIsImV4cCI6MTc1MDEzNjA0NSwiaWF0IjoxNzQ3NTQ0MDQ1LCJpc3MiOiJkZWVwZXJfaHViIiwianRpIjoiMzEwOGMzNnRrMTNwMWRhNDNjMDAwMDAyIiwibmJmIjoxNzQ3NTQ0MDQ1LCJ0eXAiOiJyZWZyZXNoIiwidXNlcl9pZCI6bnVsbH0.462r3ini7zSnk1tlIzVjo0Ww8NoQ-wj443YnPDOW_p8', 'expires_in': 3600}]
2025-05-18 01:54:05,475 - DeeperHubClient - INFO - Resposta de sucesso: auth.refresh.success [payload={'access_token': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJkZWVwZXJfaHViX3dlYnNvY2tldCIsImV4cCI6MTc0NzU0NzY0NSwiaWF0IjoxNzQ3NTQ0MDQ1LCJpc3MiOiJkZWVwZXJfaHViIiwianRpIjoiMzEwOGMzNzllaGpwMGRpYmdvMDAwMDEyIiwibmJmIjoxNzQ3NTQ0MDQ1LCJyZW1lbWJlcl9tZSI6ZmFsc2UsInNlc3Npb25faWQiOm51bGwsInR5cCI6ImFjY2VzcyIsInVzZXJfaWQiOm51bGx9.em2zxuegvNf5NXJiApY1zLXDC7YaUCfkvfLNDpkjw5I', 'refresh_token': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJkZWVwZXJfaHViX3dlYnNvY2tldCIsImV4cCI6MTc1MDEzNjA0NSwiaWF0IjoxNzQ3NTQ0MDQ1LCJpc3MiOiJkZWVwZXJfaHViIiwianRpIjoiMzEwOGMzN2E5OXJwMGRpYmdvMDAwMDIyIiwibmJmIjoxNzQ3NTQ0MDQ1LCJyZW1lbWJlcl9tZSI6ZmFsc2UsInNlc3Npb25faWQiOm51bGwsInR5cCI6InJlZnJlc2giLCJ1c2VyX2lkIjpudWxsfQ.LXouCP15JpBHTJ4hoJi8_sJLdpCwpTG1Alp0XCqChvE', 'expires_in': 3600}]
2025-05-18 01:54:06,124 - DeeperHubClient - INFO - Resposta de sucesso: user.list.success [payload={'count': 2, 'users': [{'id': '0835c858-2e05-4076-af8d-23a215dab4b2', 'username': 'romeu', 'email': 'as@asas.as', 'is_active': True, 'inserted_at': '2025-05-18T03:18:54.004000Z', 'updated_at': '2025-05-18T03:18:54.004000Z'}, {'id': '7def24c9-f718-4bbd-a5bf-b8d1f5b19ac5', 'username': 'test_user_2524c2ab', 'email': 'test_2524c2ab@example.com', 'is_active': True, 'inserted_at': '2025-05-18T04:54:04.809000Z', 'updated_at': '2025-05-18T04:54:04.809000Z'}]}]
2025-05-18 01:54:06,227 - DeeperHubClient - CRITICAL - Conexão fechada durante o envio da mensagem: received 1011 (internal error); then sent 1011 (internal error)
2025-05-18 01:54:06,228 - DeeperHubClient - ERROR - Exceção em send_message: received 1011 (internal error); then sent 1011 (internal error)
2025-05-18 01:54:06,230 - DeeperHubClient - CRITICAL - Conexão fechada durante o envio da mensagem: received 1011 (internal error); then sent 1011 (internal error)
2025-05-18 01:54:06,231 - DeeperHubClient - ERROR - Exceção em send_message: received 1011 (internal error); then sent 1011 (internal error)
2025-05-18 01:54:06,233 - DeeperHubClient - CRITICAL - Conexão fechada durante o envio da mensagem: received 1011 (internal error); then sent 1011 (internal error)
2025-05-18 01:54:06,234 - DeeperHubClient - ERROR - Exceção em send_message: received 1011 (internal error); then sent 1011 (internal error)
2025-05-18 01:54:06,236 - DeeperHubClient - INFO - Conexão fechada com sucesso
